src/app/core/data/access-status-data.service.ts

```diff
@@ -29,7 +30,15 @@ export class AccessStatusDataService extends BaseDataService<AccessStatusObject>
    * Returns {@link RemoteData} of {@link AccessStatusObject} that is the access status of the given item
    * @param item Item we want the access status of
    */
-  findAccessStatusFor(item: Item): Observable<RemoteData<AccessStatusObject>> {
+  findItemAccessStatusFor(item: Item): Observable<RemoteData<AccessStatusObject>> {
     return this.findByHref(item._links.accessStatus.href);
   }
+
+  /**
+   * Returns {@link RemoteData} of {@link AccessStatusObject} that is the access status of the given bitstream
+   * @param bitstream Bitstream we want the access status of
+   */
+  findBitstreamAccessStatusFor(bitstream: Bitstream): Observable<RemoteData<AccessStatusObject>> {
```

Comment:
This new method doesn't feel all that useful to me, it can be replaced by a `LinkService` call and `Bitstream.accessStatus`